### DISCUSSION TITLE
无法正常打开PCL，打开后出现图标后消失

### DISCUSSION THREAD

**Discussion by QAQlinmiao** [Meloong-Git/PCL#3920] [提问] [answered]
之前打开PCL一切正常，但突然有一天无法打开PCL
表现为打开后 有启动logo 过一段时间后 自己退出了
不知道是不是因为之前Xbox改时区兑换礼包的问题，电脑时区改回后就无法正常打开PCL
日志显示：
[21:53:35.794] [Start] 程序版本：Snapshot 2.7.4 (325)
[21:53:35.797] [Start] 识别码：7053-432D-8282-93A2
[21:53:35.797] [Start] 程序路径：E:\Mc\Plain Craft Launcher 2(赞助).exe
[21:53:35.797] [Start] 系统编码：System.Text.DBCSCodePageEncoding (936, GBK=True)
[21:53:35.807] [Start] 第一阶段加载用时：109 ms
然后没了……
![image](https://github.com/Hex-Dragon/PCL2/assets/83525912/8336ec69-a0d1-42d5-bb32-9b1021e98e1f)

**Comment by wuliaodexiaoluo** [comment]
试试重新下一份？

**Reply by QAQlinmiao** [reply]
> 试试重新下一份？

谢谢你的回答

我刚刚知道问题出在哪了，原来是我把程序加上了中文名导致打不开xwx

因为我尝试下载一份是可以正常打开的 只有文件名不同xwx（很难想象这个问题的出现方式是这样的

**Reply by allMagicNB** [reply]
> > 试试重新下一份？
> 
> 谢谢你的回答
> 
> 我刚刚知道问题出在哪了，原来是我把程序加上了中文名导致打不开xwx
> 
> 因为我尝试下载一份是可以正常打开的 只有文件名不同xwx（很难想象这个问题的出现方式是这样的

……下载一份？不应该是更新一份？

**Reply by QAQlinmiao** [reply]
> > > 试试重新下一份？
> > 
> > 
> > 谢谢你的回答
> > 我刚刚知道问题出在哪了，原来是我把程序加上了中文名导致打不开xwx
> > 因为我尝试下载一份是可以正常打开的 只有文件名不同xwx（很难想象这个问题的出现方式是这样的
> 
> ……下载一份？不应该是更新一份？

他的意思应该是让我重新下一份看看会不会出现问题，然后再让我更新到最新版看看会不会复现问题
但我发现最新版是可以正常打开的
然后我就怀疑到2个程序的名字问题，因为之前为了区分赞助版和非赞助版，所以加了中文才导致了上述问题的出现。（真是一个奇怪的问题x）

**Comment by Silverteal** [comment, marked as answer]
文件名中使用半角括号`(`或`)`在非常容易导致问题，请避免使用。

PCL支持中文，中文应该不是导致这个问题的直接原因。但是仍然提倡避免在文件名中使用中文。

**Reply by QAQlinmiao** [reply]
> 文件名中使用半角括号`(`或`)`在非常容易导致问题，请避免使用。
> 
> PCL支持中文，中文应该不是导致这个问题的直接原因。但是仍然提倡避免在文件名中使用中文。

原来是半角括号搞的鬼
